src/en/administration/first_steps.md

```diff
@@ -30,6 +30,8 @@ To be included in the list of Lemmy instances on [join-lemmy.org](https://join-l
 
 - [x] Federate with at least one instance from the list
 - [x] Have a site description and icon
+- [x] Don't have closed sign-ups
+- [x] Have at least 5 users who posted or commented at least once in the past month
 - [x] Be patient and wait the site to be updated, there's no fixed schedule for that
```

Comment:
Actually one more thing, it needs to run the latest major Lemmy version (currently 0.17.x).

It would also be good if you include a link to the [crawler](https://github.com/LemmyNet/lemmy-stats-crawler) which is used to build the list. And a link to [this file](https://github.com/LemmyNet/joinlemmy-site/blob/main/recommended-instances.json) where recommendations are defined.

Reply:
Implemented 👍